src/components/Chat/InputMessageV2.tsx

```diff
@@ -415,7 +425,9 @@ const InputMessageV2 = ({
         />
         {isShowToggle && (
           <ToggleSwitch
-            labelLeft={!inputFocus ? 'Anonymity' : undefined}
+            labelLeft={
+              !inputFocus ? (isAnonimityEnabled ? 'Anonymity' : 'Anonymity disabled') : undefined
+            }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The label for the toggle switch now changes based on whether anonymity is enabled or not. This is a good UX improvement as it provides clear feedback to the user about the current state of the feature. However, consider externalizing these string literals ('Anonymity', 'Anonymity disabled') to a constants file or using i18n library for better manageability and localization support in the future.


<!-- This is an auto-generated comment by OSS CodeRabbit -->